src/NossoMercadoLivreAPI.Domain/Interfaces/UnitOfWork/IUnitOfWork.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace NossoMercadoLivreAPI.Domain.Interfaces.UnitOfWork
+{
+    public interface IUnitOfWork
```

Comment:
*Usamos tudo que conhecemos que está pronto. Só fazemos código do zero se for estritamente necessário.*

Algo similar deve ser possível através do orm e já deve estar pronto. Sugiro remover.